sdks/python/src/opik/evaluation/tasks_scorer.py

```diff
@@ -15,37 +15,40 @@
 
 
 def _score_test_case(
-    test_case_: test_case.TestCase, scoring_metrics: List[base_metric.BaseMetric]
+    test_case_: test_case.TestCase,
+    scoring_metrics: Optional[List[base_metric.BaseMetric]],
```

Comment:
If at the beginning of `evaluate` you do 
```python
if scoring_metrics is None:
    scoring_metrics = []
```
You'll likely don't need to make any other changes in `evaluator.py` or `tasks_scorer.py` :) 
Making the code capable of processing empty collections properly is usually a better way than dragging these `None` checks and type hints everywhere.

Reply:
here, i thought that it would be nice to have the support of `scoring_metrics` being unset on a low level but i'll fix it sure